src/tests/sorting.spec.js

```diff
@@ -0,0 +1,40 @@
+const { expect } = require('@playwright/test');
+const { test } = require('../fixture');
+const { isSortedAlphabetically, isSortedByPrice } = require('../utils/sorting');
+const { standardUser } = require('../../config/credentials');
+
+test.describe('Sorting the inventory items', () => {
+    [
+        ['Name (A to Z)', 'az', isSortedAlphabetically, false],
+        ['Name (Z to A)', 'za', isSortedAlphabetically, true],
+    ].forEach(([optionText, sortValue, sortFunction, isDescending]) => {
```

Comment:
<3

Reply:
You could just store sortFunction inside since its the same. Or this could be useful in case you merge both tests into the one.